naucse/notebook_util.py

```diff
@@ -0,0 +1,43 @@
+from nbconvert import HTMLExporter
+from nbconvert.filters.highlight import Highlight2HTML, _pygments_highlight
+import nbformat
+import traitlets
+import pygments
+
+from . markdown_util import convert_markdown
+
+
+class Highlight2HTMLCodehilite(Highlight2HTML):
+    '''We need to do all this just to add the codehilite cssclass'''
```

Comment:
Also, as things are in this PR, highlighting doesn't work properly for IPython extensions. See the `! python -m pip install` command in the Pandas lesson.